### PR TITLE
Kiki joe/sing upcreatespace

### PR DESCRIPTION
**Pull request by aulus-plautius**
A signed in user can create a space from the host listings page.

We removed the create space route in the space_routes.py file as it is no longer needed.

### CODE REVIEW

**Review by UnisM1 — APPROVED** [review]

File: routes/host_routes.py [hunk 1]
```diff
@@ -3,18 +3,39 @@
...
+    @app.route("/create-space", methods=["POST"])
+    def create_new_space():
+        connection = get_flask_database_connection(app)
+        repository = SpaceRepository(connection)
+        space = Space(
+            None,
+            request.form["name"],
+            request.form["description"],
+            request.form["price_per_night"],
+            request.form["availability_start"],
```
Comment (MyGollyGosh) [inline routes/host_routes.py:36]: Should we add some logic to ensure dates are passed in in the correct format here?

**Review by MyGollyGosh — COMMENTED** [review]
As comment. Should we just add some logic to ensure the dates are in the right format to our database

**Review by Alexia-May — APPROVED** [review]

File: routes/host_routes.py (hunk 1 above)
Comment (aulus-plautius) [reply routes/host_routes.py:36]: We can set the input type in html to date so it you have to choose a date from a calender, ive updated it and will push the changes now

Comment (MyGollyGosh) [reply routes/host_routes.py:36]: Perfect. When that's there I'll approve =]

**Review by MyGollyGosh — APPROVED** [review]

**Pull request merged**